src/engraving/internal/engravingconfiguration.cpp

```diff
@@ -105,7 +106,14 @@ void EngravingConfiguration::init()
 
     settings()->setDefaultValue(DYNAMICS_APPLY_TO_ALL_VOICES, Val(true));
 
-    settings()->setDefaultValue(FORMATTING_COLOR, Val(Color("#A0A0A4").toQColor()));
+    settings()->setDefaultValue(FRAME_COLOR, Val(Color("#A0A0A4").toQColor()));
+    settings()->setDescription(FRAME_COLOR, muse::trc("engraving", "Frame color"));
+    settings()->setCanBeManuallyEdited(FRAME_COLOR, true);
+    settings()->valueChanged(FRAME_COLOR).onReceive(nullptr, [this](const Val& val) {
+        m_frameColorChanged.send(Color::fromQColor(val.toQColor()));
+    });
+
+    settings()->setDefaultValue(FORMATTING_COLOR, Val(Color("#C31989").toQColor()));
```

Comment:
I think that's fine for now (it may not be bad, in principle, if people can retain their previous setting, especially given that the change is quite drastic). But we can give it more thought later